external/ilasm/ilasm.depproj

```diff
@@ -6,6 +6,22 @@
     <NugetRuntimeIdentifier>$(RuntimeOS)-x64</NugetRuntimeIdentifier>
     <OutputPath>$(ToolsDir)ilasm</OutputPath>
     <IsRuntimeAssembly>false</IsRuntimeAssembly>
+    <RidSpecificAssets>true</RidSpecificAssets>
   </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NETCore.Platforms">
+      <Version>2.0.0-$(CoreFxExpectedPrerelease)</Version>
```

Comment:
I have a feeling that this will eventually break us in the future. Instead we might consider using the actual nuget package version in a property, instead of just the prerelease. This is doable through dependencies.props. cc: @davisgoodin